package.json

```diff
@@ -77,12 +83,15 @@
     "ng2-pdf-viewer": "~5.2.0",
     "ngx-bootstrap": "^3.0.0",
     "ngx-cookie": "^4.0.2",
-    "ngx-file-uploader": "git+https://github.com/enyachoke/ngx-file-uploader.git#0.0.9",
+    "ngx-file-uploader": "0.0.14",
+    "ngx-formentry": "^2.8.0",
     "ngx-mydatepicker": "^2.4.9",
-    "ngx-openmrs-formentry": "git+https://github.com/AMPATH/ngx-openmrs-formentry.git#v2.12.2",
+    "ngx-openmrs-formentry": "^2.12.6",
     "ngx-pagination": "^3.1.1",
     "ngx-select": "^0.0.1",
     "ngx-toastr": "^9.1.1",
+    "ngx-webcam": "^0.2.5",
+    "npm": "^6.9.0",
```

Comment:
Is this a Dev dependency or production code dependency? If Dev, move it to devDependencied:{}